std_msgs/Header header
l22_aero_vision/ColorRectMarker[] markers